css/_videolayout_default.scss

```diff
@@ -507,6 +518,20 @@
     width: auto;
 }
 
+#videoNotAvailableScreen {
+    text-align: center;
+    & > #avatarContainer {
```

Comment:
Are the child selectors necessary? It's a preference of mine, and not something I ask others to do, that css selectors be kept as short as possible, probably because I'm used to BEM. child selectors bind the styling a bit to the implementation of the html.

Reply:
Do you mean to extract #avatarContainer and #avatar from #videoNotAvailableScreen?

Reply:
I mean do you need >.

Reply:
I think I can remove '>' it will work too. I'll try it!